subcommand/server-acl-init/command_test.go

```diff
@@ -1153,6 +1360,8 @@ func TestRun_AnonPolicy_IgnoredWithReplication(t *testing.T) {
 				"-server-address", strings.Split(serverAddr, ":")[0],
 				"-server-port", strings.Split(serverAddr, ":")[1],
 				"-resource-prefix=" + resourcePrefix,
+				"-server-address", strings.Split(serverAddr, ":")[0],
+				"-server-port", strings.Split(serverAddr, ":")[1],
```

Comment:
These two flags are already defined here. Were you testing something in particular by duplicating them?

Reply:
Nope 🤦 good catch, thanks!